### DISCUSSION TITLE
How to ignore unique field?

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#5202] [Help]
I have a unique field, how can I make it ignore this field when editing a record?
`
 public static function form(Form $form): Form
    {
        return $form
            ->schema([
                Card::make()->schema([
                    Forms\Components\TextInput::make('title')->name('Название_категории')->required()->rule('string'),
                    Forms\Components\TextInput::make('sorting')->label('Порядок вывода на странице')->name('sorting')->required()->rule('integer')->unique(),
                ])
            ]);
    }
`

**Comment by danharrin** [comment]
https://filamentphp.com/docs/2.x/forms/validation#unique
